pipeline/tag-release-pipeline.yaml

```diff
@@ -249,3 +249,42 @@ spec:
             resource: s2i-thoth
       runAfter:
         - tag-release-run
+    if not multiple environments
+
+
+    - name: run-tags-multi-delivery
+      taskRef:
+        name: environments-pipeline
+      params:
+        - name: git_ref
+          value: pr-$(params.pr_number)
+        - name: repo_name
+          value: $(params.pr_repo)
+        - name: registry
+          value: $(tasks.pr-configuration.results.registry)
```

Comment:
These params wouldn't be available on pipeline level.
we would be extracting them in the task. 
please remove them.